Documentation/Signoff/Vehicle Lights.md

```diff
@@ -0,0 +1,133 @@
+# Vehicle Lights signoff
+## Function of subsystem 
+The function of this subsystem is to demonstrate Electrical and Computer Engineering concepts through physical means using the car's lighting system. This will allow users to interact with the car's lights and see the physical changes of the lights based on the user's input.  
+
+## Constraints
+Table. 1 Constraints
+|Description| Constraint | Source |
+|-|-|-|
+| Variable Type Range |The Arduino mega 2560 in this subsystem shall convert any ASCII variable type that ranges from 0 - 1,000,000 into a numeric variable type for better precision.| [2] C++ data types, https://www.w3schools.com/cpp/cpp_data_types.asp  |
+|Multiplexer Voltage Safety |The Multiplexer shall take 5 volts with a 10% tolerance, no more or less than that for this subsystem to prevent any damage to the multiplexer and ensure safe and sufficient operation.| Datasheet Requirement |
+|Current Safety |The LEDs in this subsystem shall take 15mA with a 5% tolerance, no more than that to prevent overcurrent damage and to ensure safe operation.| Datasheet Requirement  |
+| Socioeconomic | Any components and equipment within this subsystem should not exceed $55 dollars and the subsystems total cost should not exceed $175 to help keep the project budget in the 400-900 range| Conceptual Design| 
```

Comment:
This isn't a socioeconomic constraint. This is a cost savings constraint. For it to be socioeconomic, it needs to be motivated by concerns for the economy or society.